File: fix.py
import glob
import re

for fp in glob.glob('public/*.html'):
    with open(fp, 'r') as f:
        text = f.read()
    
    # regex to match parseSum(\'something\') and replace with parseSum('something')
    # literally backslash quote
    text = text.replace(r"parseSum(\'", "parseSum('")
    text = text.replace(r"\')", "')")
    
    with open(fp, 'w') as f:
        f.write(text)
print("done")

File: fix.js
const fs = require('fs');
const glob = require('fs').readdirSync('public').filter(x => x.endsWith('.html')).map(x => 'public/' + x);
glob.forEach(f => {
  let t = fs.readFileSync(f, 'utf8');
  t = t.replace(/parseSum\(\\'(.*?)\\'\)/g, "parseSum('$1')");
  fs.writeFileSync(f, t);
});
console.log("Done");
